app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    _, copied, file_to_copy = command.split(" ")
```

Comment:
no need to use space inside split method, it uses space by default, also use underscore only u won't use this variable later